internal/core/repository/src/main/kotlin/com/recco/internal/core/repository/RecommendationRepository.kt

```diff
@@ -23,12 +24,13 @@ import javax.inject.Inject
 class RecommendationRepository @Inject constructor(
     private val api: RecommendationApi
 ) {
-    private val supportedContentTypes = listOf(ContentTypeDTO.ARTICLES)
+    private val supportedContentTypes = listOf(ContentTypeDTO.ARTICLES, ContentTypeDTO.QUESTIONNAIRES)
 
     private val sectionsPipelines = mapOf(
         FeedSectionType.PHYSICAL_ACTIVITY_RECOMMENDATIONS to PipelineStateAware {
             api.getTailoredRecommendationsByTopic(TopicDTO.PHYSICAL_ACTIVITY, supportedContentTypes).unwrap()
                 .map(AppUserRecommendationDTO::asEntity)
+                .sortedBy { it.type != ContentType.QUESTIONNAIRE }
```

Comment:
@saulmm as far as I know, we should keep the order in which the rec-engine delivers the recommendations, which includes articles and questionnaires, do you know otherwise?